### ISSUE TITLE
How long did you train to get the result shown in the paper

### ISSUE THREAD

**Issue by junkaiman** [ligoudaner377/font_translator_gan#6]
Besides, we would gratefully appreciate it if you could also release the pre-trained model.

**Comment by ligoudaner377** [comment]
Hi, @junkaiman 
Sorry for the late response, I'm 100% occupied by another project now.

We train the model 20 epochs (10 epochs with a constant learning rate and 10 epochs with a linear decay learning rate).
With a consumer-grade GPU, the training may take 1-2 days. 

I'm sorry that this project was done at my previous University, and I put some large files (including models) on that server locally. I'll retrain it, and upload the model someday...but I'm busy now.

by the way, run
```
bash ./train.sh
```
should get the same (at least similar) performance in the paper.

**Comment by junkaiman** [comment]
thanks for your great work! we've trained for 10 epochs and i'm posting it [here](https://drive.google.com/drive/folders/1ZyPPKPNFOzFk60gYgkgh-1US_6zBa1FJ?usp=sharing) for anyone needed.

**Comment by bucaiLi** [comment]
> thanks for your great work! we've trained for 10 epochs and i'm posting it [here](https://drive.google.com/drive/folders/1ZyPPKPNFOzFk60gYgkgh-1US_6zBa1FJ?usp=sharing) for anyone needed.

你好！请问这个预训练模型下载之后放在那里？ 文件结构是什么样的
